# TCGA_Microbial_Content
Comprehensive analysis of microbial content in whole-genome sequencing samples from The Cancer Genome Atlas project

## Supplentary Tables Captions

1. Table S1. Raw counts of 5,734 TCGA WGS samples classified at species level against Microbial2023 database, excluding eukaryotes. This resulted in 11,349 species that have non-zero counts.
2. Table S2. Normalized counts of TableS1, by counts per million reads sequenced.
3. Table S3. Normalized counts of TableS1, by reads per kilobase of genome per million reads sequenced.
4. Table S4. Raw counts of 5,734 TCGA WGS samples classified at species level against Fungi_RefSeq database. This resulted in 557 species that have non-zero counts.
5. Table S5. Normalized counts of TableS4, by counts per million reads sequenced.
6. Table S6. Normalized counts of TableS4, by reads per kilobase of genome per million reads sequenced.
7. Table S7. TCGA Study Abbreviations
8. Table S8. Raw counts of TCGA WGS samples classified at genus level against Microbial2023 database, matched with Poore et al. study. This resulted in 4,550 samples and 3,561 genera.
9. Table S9. Raw counts from Poore et al. study at genus level, matched with this study. This resulted in 4,550 samples and 3,561 genera.
10. Table S10. Raw counts of TCGA WGS samples classified at species level against Fungi_RefSeq database, matched with Narunsky-Haziza et al. study. This resulted in 4,271 11. samples and 557 species.
11. Table S11. Raw counts from Narunsky-Haziza et al. study at species level, matched with this study. This resulted in 4,271 samples and 557 species.
12. Table S12. TCGA metadata, along with Hopkins IDs, Knight IDs, and Straussman IDs for each of the 5,734 samples.
13. Table S13. List of species in the Microbial2023 database, with RefSeq accessions.
14. Table S14. List of species in the Fungi database, with RefSeq accessions.
15. Table S15. RefSeq v200 name conversion to RefSeq v220, for 14 of the 557 fungal species.
